TCMSApp/src/client/app/defects/add-defect.controller.spec.js

```diff
@@ -0,0 +1,54 @@
+/* jshint -W117, -W030 */
+describe('Tests Add-Defects Open Model', function() {
```

Comment:
the thing is that I don't see here any test of "Open Modal". 
You're just declaring unused controllerModal and injecting '$uibModal'. Or I can be wrong: but nevertheless: how `Tests Add-Defects Open Model` conceptually differs from `Tests Defects`